.github/workflows/check_tz_releases.yml

```diff
@@ -0,0 +1,31 @@
+name: Check if tz git submodule is up to date
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
if it doesnt run as part of a pr how are we notified of failure?

Reply:
Runs once a week on a schedule. As proposed in #4508, we can add the badge to the readme, so people can see it too.

Reply:
Next PR can do that, also bumping the release which shouldn’t fail anything anyways. You ok with that?

Reply:
Ah sorry Chris already explained that:)

Reply:
> also bumping the release which shouldn’t fail anything anyways. You ok with that?

you are asking about doing a new release? its fine with me

Reply:
No sorry wasn’t clear: I’m asking if you’re ok with having the status badge if this workflow in the readme.

Reply:
i dont mind if its in the readme but i do think its not all that noticable there. do you actually check the readme, i certainly dont hahaha.

it would be cool if did an http post to the github api to add a comment to a never-closed github timezone update issue whenever it fails. then we'd all get a notification. the github api is very trivial to work with the only question i have is how to get perms into the gha runner to do it. surely github has info on this

Reply:
ah ok yeah its trivial too: https://docs.github.com/en/actions/security-guides/automatic-token-authentication

i would just forget the badge and modify the script to curl to the github api a post payload to a timezone update issue when it fails and do nothing when it passes. that would be best imho

Reply:
sure!